pcsx2/Config.h

```diff
@@ -1199,8 +1199,10 @@ struct Pcsx2Config
 	struct McdOptions
 	{
 		std::string Filename; // user-configured location of this memory card
+		std::string LoadedFilename; // user-configured location of this memory card
```

Comment:
I don't like this, or the variable below being in config. It's going to get lost when any setting is changed.

The only time the filename will change is in `VMManager::CheckForMemoryCardConfigChanges()` (or whatever I named the function). So you could handle it there.

I guess `ReqReindex` could go into file scope too.